superset-frontend/src/components/Select/Select.tsx

```diff
@@ -89,6 +89,12 @@ export interface SelectProps extends PickedSelectProps {
    * False by default.
    * */
   allowNewOptions?: boolean;
+  /**
```

Comment:
This comment should be generic. When you say something like `to prevent dropdown movement onscroll` you are limiting its uses.

You could use the Ant Design docs for this prop:

> Parent Node to which the selector should be rendered.